sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_service_client_async.py

```diff
@@ -417,10 +419,11 @@ def list_containers(
             The container to delete. This can either be the name of the container,
             or an instance of ContainerProperties.
         :type container: str or ~azure.storage.blob.ContainerProperties
-        :param ~azure.storage.blob.BlobLeaseClient lease:
+        :param lease:
```

Comment:
This should be `keyword` lease?

Reply:
It's actually a parameter for delete_container()